scripts/snakenet-migration/index.js

```diff
@@ -325,6 +337,53 @@ const sendAndWaitFinalization = ({from, tx, printEvents = []}) => new Promise(re
         log(status)
     }));
 
+const validate_triple = async (source_url, target_url) => {
+    log("Validating triple balances")
+    const api = await createClient(source_url);
+    const target_api = await createClient(target_url);
+
+    const assertBalances = async (address, balance) => {
+        const account = await target_api.query.system.account(address);
+        const bal = account.data.free;
+        const tripled = balance.imuln(3);
+        assert( bal.eq(tripled), `Incorrect amount for ${address}`);
+    }
+
+    let balances = [];
+
+    await api.query.system.account.entries().then( accounts => {
+        accounts.map( ([key, {data}]) => {
+            const [address] = key.toHuman()
+            //Skip Alice - only for testing purposes
```

Comment:
use the same **excluded** const

Reply:
Yes, at this time , there was not yet the excluded list. Will get updated.